components/tikv_util/src/logger/file_log.rs

```diff
@@ -5,6 +5,8 @@ use std::fs::{self, File, OpenOptions};
 use std::io::{self, ErrorKind, Write};
 use std::path::{Path, PathBuf};
 
+const BYTES_PER_MIB: usize = 1 << 20;
```

Comment:
If we use `ReadableSize`, this can be removed.

Reply:
Done